lib/sidekiq/cron/job.rb

```diff
@@ -539,19 +543,20 @@ def last_time now = Time.now.utc
         parsed_cron.previous_time(now.utc).utc
       end
 
-      def formated_enqueue_time now = Time.now.utc
+      def formatted_enqueue_time now = Time.now.utc
         last_time(now).getutc.to_f.to_s
       end
 
-      def formated_last_time now = Time.now.utc
+      def formatted_last_time now = Time.now.utc
         last_time(now).getutc.iso8601
       end
 
       def self.exists? name
-        out = false
-        Sidekiq.redis do |conn|
-          out = conn.public_send(REDIS_EXISTS_METHOD, redis_key(name))
+        out = Sidekiq.redis do |conn|
+          conn.public_send(REDIS_EXISTS_METHOD, redis_key(name))
         end
+        out = false if out == 0
+        out = true if out == 1
         out
```

Comment:
```suggestion
        1 == Sidekiq.redis do |conn|
          conn.public_send(REDIS_EXISTS_METHOD, redis_key(name))
        end
```

Reply:
That'll resolve as false when the method returns a bool (`1 == true` is `false`), which it does before Sidekiq 7

Reply:
Oooooh sorry then. I didn't know that it could return a bool. What about saving the result in some variable and then `var == true || var == 1`?